server/src/constants.ts

```diff
@@ -21,12 +21,13 @@ export const VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT_ENV_NAME =
   'VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT';
 export const VV_COSMOS_DB_CONNECTION_STRING = 'VV_COSMOS_DB_CONNECTION_STRING';
 export const VV_COSMOS_DB_NAME = 'VV_COSMOS_DB_NAME';
-export const VV_AUTO_START_TRANSCRIPTION = 'VV_AUTO_START_TRANSCRIPTION';
+export const VV_TRANSCRIPTION_BEHAVIOR = 'VV_TRANSCRIPTION_BEHAVIOR';
 export const VV_SERVER_WEBSOCKET_URL = 'VV_SERVER_WEBSOCKET_URL';
 export const VV_SERVER_WEBSOCKET_PORT = 'VV_SERVER_WEBSOCKET_PORT';
 export const VV_SERVER_HTTP_URL = 'VV_SERVER_HTTP_URL';
 export const VV_COGNITIONAPI_KEY = 'VV_COGNITIONAPI_KEY';
```

Comment:
just a note we should move off these to Azure Credentials authed through managed identity

Reply:
this is needed in order to hit the language service API in the headers of the request so I don't think we can get around this for now. we will hopefully be able to remove this when CallAutomation releases the transcription and Summary API's as part of their package.